app/controllers/messages/ViewMessageController.scala

```diff
@@ -63,9 +66,17 @@ class ViewMessageController @Inject()(mcc: MessagesControllerComponents,
   def removeMessageAndRedirectToDraftMovement(ern: String, uniqueMessageIdentifier: Long): Action[AnyContent] =
     authorisedDataRequestAsync(ern) { implicit request =>
       getMessagesService.getMessage(ern, uniqueMessageIdentifier).flatMap {
-        case Some(msg) if msg.errorMessage.exists(_.relatedMessageType.contains("IE815")) =>
-          //TODO: delete message (ETFE-2855) then route to 'revive' draft movement controller on CaM
-          Future(Redirect(testOnly.controllers.routes.UnderConstructionController.onPageLoad()))
+        case Some(msg) =>
+          msg.errorMessage match {
+            case Some(errorMessageResponse) if errorMessageResponse.relatedMessageType.contains("IE815") && errorMessageResponse.ie704.body.attributes.exists(_.lrn.isDefined) =>
+              draftMovementService.putErrorMessagesAndMarkMovementAsDraft(ern, errorMessageResponse).map {
+                case Some(draftId) => Redirect(appConfig.emcsTfeCreateMovementTaskListUrl(ern, draftId))
+                case None => InternalServerError(errorHandler.internalServerErrorTemplate)
```

Comment:
Would we want to log something here? So we've got some additional information in Kibana if this ever happens or would we have all we need?